### PR TITLE
Improving the Resource Library

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
1.  Included `/news` collection into the library.
2. Included `/events, type:workshop` collection into the library.
        - Had to change `topic` to `type` in order to filter properly.
3. Added a Recorded Webinar
4. Fixed the `type` , `technology`, `product`, `mesh` categories in some resources.
        - Will go through all collections later to add these to all remaining ones. I've covered most of them.
5. Removed some unused filters.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by debo19** [comment]
@leecalcote I've include `/news` into the resource library. Should we have a filter- `type: Events` as well? I tried including only the workshops for now. I wasn't sure about `Events` as a type here.

**Comment by l5io** [comment]
🚀 Preview for commit 6527e3d5a61387bf3c0bca73f9ef8e78d1c5e85e at: https://61f56b6500f52550c65c4271--layer5ng.netlify.app

**Comment by leecalcote** [comment]
I suppose that there is more logic supporting including Events than there is to not include events.

**Comment by l5io** [comment]
🚀 Preview for commit e5a95e3f0c37a5c511deb942ba2a9a76f7cca3ef at: https://61f78189663ebf5038fe4d5e--layer5ng.netlify.app

File: src/sections/Resources/Resources-grid/options.js [hunk 1]
```diff
@@ -4,17 +4,16 @@ export const options =  [
...
-      { id: 4, value: "FAQ", label: "FAQ" },
-      { id: 5, value: "Guide", label: "Guide" },
-      { id: 6, value: "Interview", label: "Interview" },
-      { id: 7, value: "News", label: "News" },
-      { id: 8, value: "Podcast", label: "Podcast" },
-      { id: 9, value: "Presentation", label: "Presentation" },
-      { id: 10, value: "Recorded Webinar", label: "Recorded Webinar" },
-      { id: 11, value: "Tutorial", label: "Tutorial" },
-      { id: 12, value: "White Paper", label: "White Paper" }]
+      { id: 2, value: "News", label: "News" },
```
Comment (warunicorn19) [inline src/sections/Resources/Resources-grid/options.js:7]: Any particular reason why we don't have this in alphabetical order anymore?

**Review by warunicorn19 — APPROVED** [review]
Merge at-will

File: src/sections/Resources/Resources-grid/options.js (hunk 1 above)
Comment (debo19) [reply src/sections/Resources/Resources-grid/options.js:7]: @warunicorn19 I wanted to push the "(0)"s to the end. We can switch it back later once we have each of those collections displayed.

**Pull request merged**